_includes/head.html

```diff
@@ -1,6 +1,11 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:title" content="{{ site.baseurl }}/{{ post.title }}" />
```

Comment:
We shouldn't need `site.baseurl` here right? We just want the `post.title`

Reply:
Yes, that is not required. I have removed it and pushed the fix. Also updated `twitter:image` to fetch from `post.feature-img`.